core/scheduler/src/main/scala/org/apache/openwhisk/core/scheduler/FPCSchedulerServer.scala

```diff
@@ -59,13 +59,21 @@ class FPCSchedulerServer(scheduler: SchedulerCore, systemUsername: String, syste
           logger.warn(this, "Scheduler is disabled")
           scheduler.disable()
           complete("scheduler disabled")
-        } ~ (path(FPCSchedulerServer.queuePathPrefix / "total") & get) {
-          complete {
-            scheduler.getQueueSize.map(_.toString)
+        } ~ (path(FPCSchedulerServer.queuePathPrefix) & get) {
+          pathEndOrSingleSlash {
+            complete(scheduler.getQueueStatusData.map(s => s.toJson))
+          } ~ (path("count") & get) {
+            complete(scheduler.getQueueSize.map(s => s.toJson))
           }
-        } ~ (path(FPCSchedulerServer.queuePathPrefix / "status") & get) {
-          complete {
-            scheduler.getQueueStatusData.map(s => s.toJson)
+        } ~ (path("activation" / "count") & get) {
```

Comment:
nice this will be great to have for the schedulers. One other thing we need to address for true zero downtime is kafka for the scheduler since activations could get stuck waiting on a specific scheduler topic while the scheduler is being restarted.

Reply:
Currently, that is handled in this way.

1. When a scheduler is disabled, it will first remove its endpoint from ETCD.
2. Controllers no longer send any new requests to the scheduler as there is no endpoint.
3. Still, messages in Kafka can be delivered to the scheduler because the consumer does not shut down.
4. Before the number of in-flight activation becomes 0, the scheduler is not deployed.
5. Once all activations are handled, the scheduler will be redeployed.